oneflow/core/functional/impl/activation_functor.cpp

```diff
@@ -143,6 +143,38 @@ class HardTanhGradFunctor {
   std::shared_ptr<OpExpr> op_;
 };
 
+class ReLU6Functor {
+  public:
+    ReLU6Functor() {
+      op_ = CHECK_JUST(one::OpBuilder("hardtanh").Input("in").Output("out").Build());
+    }
+    Maybe<Tensor> operator()(const std::shared_ptr<one::Tensor>& x, bool inplace) const{
+      // ASSERT_FALSE(inplace);
+      CHECK_OR_RETURN(inplace==false) << "nn.functional.relu6 does not support inplace now";
```

Comment:
```suggestion
      CHECK_OR_RETURN(!inplace) << "nn.functional.relu6 does not support inplace now";
```
bool 类型一般不用 `==` 和 `!=` 这样判断。

另外，原python版本对于 inplace 情况只是打印警告，并不是提前返回。这个怎么修改，看有其它路过的同事给建议么。

Reply:
修改好了